.bazelversion

```diff
@@ -1,4 +1,4 @@
-5.3.0
+rolling
```

Comment:
not sure about using rolling here; it might be bad for contributors if things suddenly fail due to a new release